docs/guides/triggermesh-privatelink.md

```diff
@@ -0,0 +1,133 @@
+# Creating a PrivateLink to TriggerMesh Services
+
+[PrivateLink](https://aws.amazon.com/privatelink) is a service that enables you to establish private connectivity 
+between VPCs and services hosted on Amazon Web Services (AWS) or on-premises, 
+without exposing data to the internet. Let’s look at the configuration required
+to access TriggerMesh services deployed on an AWS Elastic Kubernetes Service (AWS) 
+cluster across AWS accounts over a PrivateLink connection.
+
+## Configuring the Istio Ingress Gateway
+
+The Istio networking layer (or others such as Kourier) creates an Elastic Load 
+Balancer (ELB) allowing public access to the Knative services running in the cluster. 
```

Comment:
knative still mentioned here, maybe not problem

Reply:
👍